### PR TITLE
Use chunked delete for deleting Storage objects in purge

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: resolwe/flow/models/base.py [hunk 1]
```diff
@@ -12,6 +12,45 @@
 MAX_SLUG_RETRIES = 10
 
 
+# NOTE: This method is used in migrations.
+def delete_chunked(queryset, chunk_size=500):
```
Comment (mstajdohar) [inline resolwe/flow/models/base.py]: Why did you move the method out of the QuerySet?

Comment (mstajdohar) [reply resolwe/flow/models/base.py]: Because of migrations?

Comment (mstajdohar) [reply resolwe/flow/models/base.py]: Why didn't it work when called `DataQuerySet._delete_chunked `?

Comment (dblenkus) [reply resolwe/flow/models/base.py]: Custom queryset is not initialized in migrations. It feels less hacky to me to make a separate function than making the method static.

Comment (mstajdohar) [reply resolwe/flow/models/base.py]: > Custom queryset is not initialized in migrations

Thank you for the explanation.
